src/ImageManipulation.php

```diff
@@ -871,6 +871,8 @@ function (AssetStore $store, $filename, $hash, $variant) use ($callback) {
                         $tuple = $store->setFromStream($result->getStream(), $filename, $hash, $variant);
                         return [$tuple, $result];
                     } finally {
+                        // Unload the Intervention Image resource so it can be garbaged collected
+                        $res = $backend->setImageResource(null);
```

Comment:
This bit improves memory usages when generating lots of thumbnails with the GD image driver.

It will however alter the behaviour everywhere else, so you can't test it just on the migration script.